opendebates/templates/opendebates/list_ideas.html

```diff
@@ -71,7 +71,7 @@
         </option>
       </select>
       <div class="display-count">
-        Displaying <strong>{{ ideas|length }} Questions</strong>
+        Displaying <strong>{% cache 30 idea_count %}{{ ideas.count }}{% endcache %} Questions</strong>
```

Comment:
I might be misreading here, but doesn't this need search_term and category in the cache key?  (This template is used on the category + search + category-search views in addition to the homepage)